doc/manual/rl-next/github-nested-org-urls.md

```diff
@@ -0,0 +1,22 @@
+---
+synopsis: "GitLab flakerefs now allow for nested organisations"
+prs: 9163
+---
+
+GitLab allows to nest organisations (`gitlab.com/org/suborg/subsuborg/repo`).
+The flakeref URL for GitLab repositories now supports that.
+
+This means that it isn't possible any more to use the `/<ref>` (or `/<rev>`) syntax for GitLab repositories (as that would be ambiguous), and users need to pass an explicit query parameter (`?ref=<ref>` or `?rev=<rev>`).
```

Comment:
There's precedent in other tools for `//<ref>`.
If we don't implement that, I think we should explicitly deny it, to avoid confusion and perhaps allow it later.
Relevant notes: https://github.com/NixOS/nix/pull/8773#issuecomment-1689436944

Less critically, we may also consider supporting `/<rev>`, as full `rev`s are easily distinguished by their length and hex characters.